static/src/stylesheets/_vars.scss

```diff
@@ -136,6 +136,18 @@ $sport-away:             #003f64;
 
 // Stati
```

Comment:
Small point, and not related to Pat's changes, but this isn't actually the correct pluralisation of "status" in Latin or otherwise. If we insist on using Latinate declension (wrongly - we only use foreign-language inflections for loan-words we'd encase in _italics_), then the genitive case of `status` is `statōrum`.